pkg/k8s/k8s.go

```diff
@@ -59,11 +61,78 @@ func CmdAddK8s(ctx context.Context, args *skel.CmdArgs, conf types.NetConf, epID
 		"Namespace":        epIDs.Namespace,
 	})
 
+	d, err := dataplane.GetDataplane(conf, logger)
+	if err != nil {
+		return nil, err
+	}
+
 	logger.Info("Extracted identifiers for CmdAddK8s")
+	if runtime.GOOS == "windows" {
```

Comment:
Rather than stubbing out several functions, why not replace this entire if block with `err := utils.CheckForSpuriousAdd()` and then have a since Windows/Linux-specific function?  I think that'd be a lot cleaner.

Reply:
Will do it with another PR.